manifests/2.6.0/opensearch-2.6.0.yml

```diff
@@ -32,6 +32,15 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version
+  - name: ml-commons
+    repository: https://github.com/opensearch-project/ml-commons.git
+    ref: tags/2.6.0.0
```

Comment:
We don't have 2.6 tag yet. Change to `ref: '2.x'` , check line 28

Reply:
resolved.